File: backend2/DataAccess/DaoGenerated/システム共有/tm_affileDto.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: DTO定義
//             ファイルマスタ
// 作成日　　: 2024.07.23
// 作成者　　: 
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_affileDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_affile";
		public string siyokbn { get; set; }                       //使用区分
		public string filenm { get; set; }                        //ファイル名
		public int filetype { get; set; }                         //ファイルタイプ
		public byte[] filedata { get; set; }                      //ファイルデータ
    }
}

File: GJSBackend/BusinessService/Db/DaoGenerated/システム共有/tm_affileDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             ファイルマスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tm_affileDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_affile"
        Public Property siyokbn As String                       '使用区分
        Public Property filenm As String                        'ファイル名
        Public Property filetype As Integer                         'ファイルタイプ
        Public Property filedata As Byte()                      'ファイルデータ
    End Class
End Namespace
